packages/core/botpress-builtins/src/elements/builtin_action-button.js

```diff
@@ -26,7 +26,7 @@ export default {
           {
             properties: {
               action: {
-                enum: ['Say something']
+                enum: ['Share']
```

Comment:
Why do you change it?

Reply:
Take a look at the full file diff. I added lots of new button types/options and some diff sections just look weird because of how it was merged with conflicts...